lib/sufia/engine.rb

```diff
@@ -1,4 +1,12 @@
 module Sufia
+  class Statefile
+    def self.default
+      return '/tmp/minter-state' if Rails.env.development? || Rails.env.test?
+      raise NotImplementedError, "can't use the default statefile for production" unless Dir.exist?('/var/sufia')
```

Comment:
I believe the solution in the ticket was to make this change in the Sufia initializer (within the application) vs. in the Sufia engine:

https://github.com/projecthydra/sufia/issues/1532#issue-128606782

And we'd either need a generator (to run on upgrade) to automatically "fix" folks initializers, or skip the generator and include instructions in the release notes. (This latter part is already covered in #1532.)